doc/install_linux_prerequisites.md

```diff
@@ -93,6 +93,6 @@ Python 2.7.8 should be already on the machine. Install Python 3.6+ if needed.
 sudo zypper refresh && sudo zypper --non-interactive install gcc libffi-devel python-devel openssl-devel
 ```
 
-### CoreOS Stable-899.15.0 / Beta-1010.1.0 / Alpha-1010.1.0
+### Flatcar
 
-Python is not installed by default.
+Python is installed in the Azure-specific distribution of Flatcar, but is installed into the non-standard location `/usr/share/oem/python/bin/python`.
```

Comment:
Given python is already installed on "Azure-specific distribution of Flatcar", shouldn't we provide the command to install Azure CLI dependencies?

Reply:
This description is provided by ahrkrak. @ahrkrak Hi, could you please help to have a look~

Reply:
Now I doubt if there is really someone to use Flatcar to install Azure CLI. 🤣